controllers/operatorpolicy_controller.go

```diff
@@ -300,16 +333,16 @@ func buildOperatorGroup(
 ) *operatorv1.OperatorGroup {
 	operatorGroup := new(operatorv1.OperatorGroup)
 
-	gvk := schema.GroupVersionKind{
-		Group:   "operators.coreos.com",
-		Version: "v1",
-		Kind:    "OperatorGroup",
+	if policy.Spec.OperatorGroup == nil {
+		operatorGroup.SetGroupVersionKind(operatorgroupgvk)
+		operatorGroup.ObjectMeta.SetName(policy.Name + "-default-og")
```

Comment:
You might run into the 63 character limit if the policy name is long. I'd suggest guarding against this.

Reply:
Would enforcing a specific length (i.e., enforce the policy name to be 63 - length of default OperatorGroup suffix) be a good approach? Another way I'm thinking of is dynamically renaming the default OperatorGroup but that might make it difficult to look for it on the cluster.

Reply:
Could you reuse the subscription name since that is a required field?